src/nora/components/Colors.css

```diff
@@ -9,4 +9,6 @@
 
   --NoraGray--opaque: #4f5972;
   --NoraGray--translucent: #eaebee;
+  --NoraGrayHeader--opaque: #7c8699;
```

Comment:
I've sent an email regarding "Nora Gray's", so I think this will need further iterating. But it's fine for now I think.